graph.go

```diff
@@ -708,6 +704,10 @@ func (g *Graph) validateTransactionParents(tx *Transaction) error {
 		if maxDepth < parent.Depth { // Update max depth witnessed from parents.
 			maxDepth = parent.Depth
 		}
+
+		if parent.Seed != tx.ParentSeeds[i] {
```

Comment:
Could you add a test for checking that validateTransactionParents validates parent seeds as expected?